core/domain/user_services.py

```diff
@@ -1971,29 +1969,29 @@ def remove_community_reviewer(user_id):
         user_community_rights_model.delete()
 
 
-def get_community_reviewer_usernames(review_type, language_code=None):
+def get_community_reviewer_usernames(review_category, language_code=None):
     """Returns a list of usernames of users who has rights to review item of
-    given review type.
+    given review category.
 
     Args:
-        review_type: str. The type of of review item. One of translation,
-            voiceover or question.
+        review_category: str. str. The category which for in which user can
+            review.
         language_code: None|str. The language code for translation or voiceover
-            review type.
+            review category.
 
     Returns:
         list(str.) A list of usernames.
     """
     reviewer_ids = []
-    if review_type == constants.REVIEW_CATEGORY_TRANSLATION:
+    if review_category == constants.REVIEW_CATEGORY_TRANSLATION:
         reviewer_ids = (
             user_models.UserCommunityRightsModel
             .get_translation_reviewer_user_ids(language_code))
-    elif review_type == constants.REVIEW_CATEGORY_VOICEOVER:
+    elif review_category == constants.REVIEW_CATEGORY_VOICEOVER:
         reviewer_ids = (
             user_models.UserCommunityRightsModel
             .get_voiceover_reviewer_user_ids(language_code))
-    elif review_type == constants.REVIEW_CATEGORY_QUESTION:
+    elif review_category == constants.REVIEW_CATEGORY_QUESTION:
```

Comment:
Error here if language_code is not None.

Reply:
Done!